Resources/Locale/en-US/guardian/guardian.ftl

```diff
@@ -0,0 +1,12 @@
+
+## Guardian host specific
+
+guardian-created = You feel.. haunted
```

Comment:
loc strings for this one should be a bit more descriptive + text definitely needs some discussion